docker-compose/hive.env

```diff
@@ -0,0 +1,23 @@
+# ConnectionURL for hive-server will be overriden, see docker-compose.yml
+HIVE_SITE_CONF_javax_jdo_option_ConnectionURL=jdbc:postgresql://database:5432/metastore
+HIVE_SITE_CONF_javax_jdo_option_ConnectionDriverName=org.postgresql.Driver
+HIVE_SITE_CONF_javax_jdo_option_ConnectionUserName=hive
+HIVE_SITE_CONF_javax_jdo_option_ConnectionPassword=hive
+HIVE_SITE_CONF_datanucleus_autoCreateSchema=false
+# metastore.uris for hive-server will be overriden, see docker-compose.yml
+HIVE_SITE_CONF_hive_metastore_uris=thrift://127.0.0.1:9083
+HIVE_SITE_CONF_hive_metastore_schema_verification=true
+HIVE_SITE_CONF_hive_execution_engine=mr
+HIVE_SITE_CONF_hive_support_concurrency=false
+HIVE_SITE_CONF_hive_driver_parallel_compilation=true
+HIVE_SITE_CONF_hive_metastore_warehouse_dir=s3a://hive/warehouse
+HIVE_SITE_CONF_hive_metastore_event_db_notification_api_auth=false
+HIVE_SITE_CONF_hive_server2_authentication=NOSASL
+# csv support from presto todo https://github.com/prestosql/presto/issues/2678#issuecomment-580255387
```

Comment:
Should be it's own issue, no?

Reply:
What do u mean here? It is presto related issue, and I see it was fixed so far. I would prefer to have ref, maybe remove todo (WDYT?)

Reply:
I was just talking about the fact that it says todo. If it is fixed we should just remove the 'todo' part of it.